Store/Store.php

```diff
@@ -117,6 +117,12 @@ public static function getConfigDefinitions()
 			'adwords.client_id'        => null,
 			// developer token used for automating ad creation
 			'adwords.developer_token'  => null,
+
+			// Braintree Payments
+			'braintree.environment' => 'sandbox',
+			'braintree.merchant_id' => null,
+			'braintree.public_key'  => null,
+			'braintree.private_key' => null,
```

Comment:
The Authorize.net config definitions never made it into the ```Store::getConfigDefinitions()``` - does it make sense to have them in Store as well, or remove these and do what we did for Authorize.net?